src/components/navigation-dropdown/navigation-dropdown.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ControlSelect from '@mapbox/react-control-select';
+// import { routeToPrefixed } from '@mapbox/batfish/modules/route-to';
+
+class NavigationDropdown extends React.Component {
+  render() {
+    const { props } = this;
+    const optionsArray = props.dropdownOptions.map(option => {
+      return {
+        label: option.title,
+        value: option.path
+      };
+    });
+    const currentOption = optionsArray.filter(option => {
+      return props.currentPath === option.value;
+    });
+
+    return (
+      <div className="py24 px24 flex-parent flex-parent--space-between-main flex-parent--center-cross">
+        <ControlSelect
+          id="navigate-this-section"
+          onChange={() => {
+            // routeToPrefixed(value);
```

Comment:
@davidtheclark suggestions on how to handle this here? I'm guessing we don't want to use something Batfish-specific like this, but let me know if it's ok!

Reply:
NavigationDropdown can have `onChange` as a prop, and pass that the value of the option that was selected.

NavigationAccordion could take an `onDropdownChange` prop that passes through. 

It's kind of awkward, but would work in this context. It's arguable that `<select>`s have some undesirable qualities for usage as a navigation dropdown — so we might want to consider building a new component for this in the future.

Reply:
I'm having a hard time understanding how information is flowing in that scenario ☝️